scripts/cmake-presets/ci-windows-github.json

```diff
@@ -39,7 +39,8 @@
       "displayName": "fast build with testing and JSON",
       "cacheVariables": {
         "CELERITAS_BUILD_TESTS": {"type": "BOOL", "value": "ON"},
-        "CELERITAS_USE_JSON": {"type": "BOOL", "value": "ON"}
```

Comment:
Is JSON support intentional removed?

Reply:
The configure option is obsolete; it's now always on and this option is ignored.